pkg/client/imagestreams.go

```diff
@@ -139,7 +140,7 @@ func transformUnsupported(err error) error {
 	// enabled by policy. A create request will return a Forbidden(403) error.
 	// We want to return ErrImageStreamImportUnsupported to allow fallback behavior
 	// in clients.
-	if apierrs.IsForbidden(err) {
+	if apierrs.IsForbidden(err) && !strings.Contains(strings.ToLower(err.Error()), "exceeded quota") {
```

Comment:
That's reasonable. So far there's just an extended test verifying this. I'll do it.